dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -18,6 +18,8 @@ import {
   SystemProgram,
   Transaction as SolanaTransaction,
 } from "@solana/web3.js";
+// @ts-ignore
```

Comment:
Why is the `@ts-ignore` directive needed here?

If needed, please use `@ts-expect-error - [explanation]` so there's context and this can later be removed if no longer a TS error since `expect-error` will then flag it again.

Reply:
use @ts-expect-error now

Reply:
Thanks, still needs an explanation please as to why there is a TS error (i.e. `@ts-expect-error - [explanation]` as shown above)

Reply:
Because tronWeb doesn't support TypeScript at this moment,  so I add this in case any error